app/views/patron_requests/new.html.erb

```diff
@@ -30,8 +33,8 @@
     </div>
 
     <% if f.object.items_in_location.one? %>
-      <div class="accordion-item my-4 shadow-sm" id="placeholder2-accordion">
-        <%= render 'accordion_header', title: 'Scan or pickup request', target: 'placeholder2' %>
+      <div class="accordion-item my-4 shadow-sm step-placeholder accordion-step-<%= scan_or_pickup_step ||= step_enum.next %>" id="scan-or-pickup-placeholder" data-patronRequest-forRequestType="none" data-patronRequest-target="accordion">
```

Comment:
<%= scan_or_pickup_step ||= step_enum.next %> should be 
<%= current_step = (scan_or_pickup_step ||= step_enum.next) %>